LICENSE.md

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Airbnb
+Copyright (c) 2018 Airbnb
```

Comment:
Technically these years don’t need to be updated at all, legally; I’d prefer to just set this back to the original year the guide was created and leave it be forever.

Reply:
What about {year}-present?